classes/Tools.php

```diff
@@ -3871,17 +3869,15 @@ public static function arrayUnique($array)
      */
     public static function cleanNonUnicodeSupport($pattern)
     {
-        $context = Context::getContext();
-        $containerFinder = new ContainerFinder($context);
-        try {
-            $container = $containerFinder->getContainer();
-            $characterCleaner = $container->get('prestashop.core.string.character_cleaner');
-        } catch (ContainerNotFoundException $e) {
-            // Used when the container is not generated
-            $characterCleaner = new CharacterCleaner();
-        }
+        @trigger_error(
```

Comment:
Is something like this ok?
```
     * @deprecated Since 8.0.0 and will be removed in the next major.
     * This function always returns $pattern argument without being modified.
     * See more: https://github.com/PrestaShop/PrestaShop/issues/27362
```

Reply:
Only the first line will be ok.

And document it in the description table.

Reply:
done